devmgmtV2/telemetry_cron.js

```diff
@@ -110,7 +110,7 @@ let generateTelemetry = (telemetryType, telemetryValue) => {
     }
     if (telemetryNow !== null) {
```

Comment:
if (!telemetryNow)

Reply:
Wouldn't that negate the logic, causing the telemetry to be "generated" when it is neither pertaining to users connected or internet connectivity?

Reply:
if(telemetry)